src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,58 @@
+import cn from 'classnames';
+
+interface Props {
+  setFilteringMode: (arg0: string) => void,
+  filteringMode: string;
+}
+
+export const TodoFooter: React.FC<Props>
+  = ({ setFilteringMode, filteringMode }) => {
+    return (
+      <footer className="todoapp__footer">
+        <span className="todo-count">
+          3 items left
+        </span>
+
+        {/* Active filter should have a 'selected' class */}
+        <nav className="filter">
+          <a
+            href="#/"
+            className={cn({
+              filter__link: true,
+              selected: filteringMode === 'all',
```

Comment:
Let's create an enum for that
```suggestion
              selected: filteringMode === FilterType.ALL,
```